### ISSUE TITLE
Misaligned DropdownMenu 

### ISSUE THREAD

**Issue by kirill-grouchnikov** [JetBrains/compose-jb#139]
Taking the core Compose sample from https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-master-dev/compose/material/material/samples/src/main/java/androidx/compose/material/samples/MenuSamples.kt and trying it on desktop (macOS + build129):

```
fun main() {
    Window(
        title = "Drop Demo",
        size = IntSize(500, 600)
    ) {
        var expanded by remember { mutableStateOf(false) }

        val iconButton = @Composable {
            IconButton(modifier = Modifier.background(Color.Green),
                onClick = { expanded = true }) {
                Icon(Icons.Default.MoreVert)
            }
        }
        DropdownMenu(
            expanded = expanded,
            onDismissRequest = { expanded = false },
            toggle = iconButton,
            toggleModifier = Modifier.fillMaxSize().wrapContentSize(Alignment.TopStart)
        ) {
            DropdownMenuItem(onClick = { /* Handle refresh! */ }) {
                Text("Refresh")
            }
            DropdownMenuItem(onClick = { /* Handle settings! */ }) {
                Text("Settings")
            }
            Divider()
            DropdownMenuItem(onClick = { /* Handle send feedback! */ }) {
                Text("Send Feedback")
            }
        }
    }
}
```

When the green button with the vertical dots is clicked, the menu is displayed all the way in the bottom right corner of the window:
<img width="612" alt="dropdown-desktop" src="https://user-images.githubusercontent.com/52874/100151073-a259b380-2e6e-11eb-981c-d9340035d46c.png">

**Comment by kirill-grouchnikov** [comment]
Relevant code seems to be in the first (and only so far) pass of https://android-review.googlesource.com/c/platform/frameworks/support/+/1502053/2/compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/DesktopPopup.kt

**Comment by olonho** [comment]
Yes, this issue is known and will be fixed with https://android-review.googlesource.com/c/platform/frameworks/support/+/1507759

**Comment by olonho** [comment]
Shall be fixed with 0.2.0-build131
![image](https://user-images.githubusercontent.com/2600522/100372639-990d4a00-301a-11eb-9922-29e1e9c6a2f0.png)

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5594">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
